keel-clouddriver/src/main/kotlin/com/netflix/spinnaker/config/ClouddriverConfiguration.kt

```diff
@@ -32,13 +32,11 @@ import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
 import org.springframework.context.ApplicationContext
 import org.springframework.context.annotation.Bean
 import org.springframework.context.annotation.Configuration
-import org.springframework.context.annotation.Import
 import retrofit2.Retrofit
 import retrofit2.converter.jackson.JacksonConverterFactory
 
 @Configuration
 @ConditionalOnProperty("clouddriver.enabled")
-@Import(RetrofitConfiguration::class)
```

Comment:
removed this where it was referenced - all it does is configure a RetrofitLogLevel bean which we don't actually use anywhere

Reply:
D'oh. Cruft from before I switched over to Retrofit 2. Good spot

Reply:
I had to create a new config key just for that bean as well since they changed one of the log level enum names in Retrofit 2.